config-rbac.yaml

```diff
@@ -12,6 +12,9 @@ rules:
   - apiGroups: [""]
     resources: ["pods", "pods/status", "pods/log", "pods/exec", "jobs", "jobs/status", "jobs/log"]
     verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: ["extensions", "apps"]
+    resources: ["deployments"]
```

Comment:
Great!

Reply:
Why it should go in a separate `apiGroups`?

Reply:
Good point, is it possilbe to have the following structure?

```

apiGroups: ["", "extensions", "apps"]
...
...
```

If yes, I can refactor.

Reply:
No idea, do you have a reference? why the `deployments` should have `"extensions", "apps"` ?

Reply:
Oh, I took it from the official k8s guide here https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-examples 

It seems like a common RBAC settings in K8s.

Reply:
Ah ok, then. and why no verbs is specified?